packages/core/src/codewhisperer/models/model.ts

```diff
@@ -311,12 +324,23 @@ export enum BuildSystem {
 
 export class ZipManifest {
     sourcesRoot: string = 'sources/'
-    dependenciesRoot: string | undefined = 'dependencies/'
+    dependenciesRoot: string = 'dependencies/'
```

Comment:
Curious why the undefined was removed here, do we upload dependencies for SQL conversion?

Reply:
Yes, for SQL conversions, that metadata ZIP file is put under the `dependencies` folder

Reply:
Would this affect non SQL conversion projects where the dependencies are uploaded?

Reply:
No, it should never be undefined anyway since the dependencies installed from `mvn install` will go there